diff --git a/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean b/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean
index dae7954..378be87 100644
--- a/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean
+++ b/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean
@@ -106,11 +106,12 @@ theorem rpow_def_of_neg {x : ℝ} (hx : x < 0) (y : ℝ) : x ^ y = exp (log x *
   · rw [Complex.ofReal_eq_zero]
     exact ne_of_lt hx
 
--- simp is called on three goals at once (leaving one), with different simp sets
-set_option linter.flexible false in
 theorem rpow_def_of_nonpos {x : ℝ} (hx : x ≤ 0) (y : ℝ) :
     x ^ y = if x = 0 then if y = 0 then 1 else 0 else exp (log x * y) * cos (y * π) := by
-  split_ifs with h <;> simp [rpow_def, *]; exact rpow_def_of_neg (lt_of_le_of_ne hx h) _
+  split_ifs with h h'
+  · simp [rpow_def, h, h']
+  · simp [rpow_def, h, h']
+  · exact rpow_def_of_neg (lt_of_le_of_ne hx h) _
 
 @[bound]
 theorem rpow_pos_of_pos {x : ℝ} (hx : 0 < x) (y : ℝ) : 0 < x ^ y := by
@@ -467,12 +468,17 @@ theorem rpow_two (x : ℝ) : x ^ (2 : ℝ) = x ^ 2 := by
 theorem rpow_neg_one (x : ℝ) : x ^ (-1 : ℝ) = x⁻¹ := by
   rw [rpow_neg_eq_inv_rpow, rpow_one]
 
--- TODO: fix non-terminal simp (acting on three goals, with different simp sets, leaving two)
-set_option linter.flexible false in
 theorem mul_rpow (hx : 0 ≤ x) (hy : 0 ≤ y) : (x * y) ^ z = x ^ z * y ^ z := by
-  iterate 2 rw [Real.rpow_def_of_nonneg]; split_ifs with h_ifs <;> simp_all
-  · rw [log_mul ‹_› ‹_›, add_mul, exp_add, rpow_def_of_pos (hy.lt_of_ne' ‹_›)]
-  all_goals positivity
+  rcases hx.eq_or_lt with rfl | hx'
+  · rcases eq_or_ne z 0 with rfl | hz
+    · simp
+    · simp [hz]
+  rcases hy.eq_or_lt with rfl | hy'
+  · rcases eq_or_ne z 0 with rfl | hz
+    · simp
+    · simp [hz]
+  rw [rpow_def_of_pos hx', rpow_def_of_pos hy', rpow_def_of_pos (mul_pos hx' hy'),
+    log_mul hx'.ne' hy'.ne', add_mul, exp_add]
 
 theorem inv_rpow (hx : 0 ≤ x) (y : ℝ) : x⁻¹ ^ y = (x ^ y)⁻¹ := by
   rw [← rpow_neg_eq_inv_rpow, rpow_neg hx]
