diff --git a/Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean b/Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean
index afeed57..30c03cf 100644
--- a/Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean
+++ b/Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean
@@ -157,8 +157,6 @@ theorem posLog_prod {α : Type*} (s : Finset α) (f : α → ℝ) :
 ## Estimates for Sums
 -/
 
--- TODO: non-terminal simp followed by positivity
-set_option linter.flexible false in
 /-- Estimate for `log⁺` of a sum. See `Real.posLog_add` for a variant involving
 just two summands. -/
 theorem posLog_sum {α : Type*} (s : Finset α) (f : α → ℝ) :
@@ -173,10 +171,12 @@ theorem posLog_sum {α : Type*} (s : Finset α) (f : α → ℝ) :
   _ = log⁺ |∑ t ∈ s, f t| := by
     rw [Real.posLog_abs]
   _ ≤ log⁺ (∑ t ∈ s, |f t|) := by
-    apply monotoneOn_posLog (by simp) (by simp [Finset.sum_nonneg])
+    apply monotoneOn_posLog (Set.mem_Ici.2 (abs_nonneg _))
+      (Set.mem_Ici.2 (Finset.sum_nonneg fun _ _ ↦ abs_nonneg _))
     simp [Finset.abs_sum_le_sum_abs]
   _ ≤ log⁺ (∑ t ∈ s, |f t_max|) := by
-    apply monotoneOn_posLog (by simp [Finset.sum_nonneg]) (by simp; positivity)
+    apply monotoneOn_posLog (Set.mem_Ici.2 (Finset.sum_nonneg fun _ _ ↦ abs_nonneg _))
+      (Set.mem_Ici.2 (Finset.sum_nonneg fun _ _ ↦ abs_nonneg _))
     apply Finset.sum_le_sum (fun i ih ↦ ht_max.2 i ih)
   _ = log⁺ (s.card * |f t_max|) := by
     simp [Finset.sum_const]
