diff --git a/Mathlib/LinearAlgebra/Determinant.lean b/Mathlib/LinearAlgebra/Determinant.lean
index 9fbd1a9..dd6d1fe 100644
--- a/Mathlib/LinearAlgebra/Determinant.lean
+++ b/Mathlib/LinearAlgebra/Determinant.lean
@@ -322,12 +322,8 @@ lemma isUnit_iff_isUnit_det [Module.Finite R M] [Module.Free R M] (f : M →ₗ[
   let b := Module.Free.chooseBasis R M
   rw [← isUnit_toMatrix_iff b, ← det_toMatrix b, Matrix.isUnit_iff_isUnit_det (toMatrix b b f)]
 
-/-- If a linear map has determinant different from `1`, then the module is free. -/
-theorem free_of_det_ne_one {f : M →ₗ[R] M} (hf : f.det ≠ 1) : Module.Free R M := by
-  by_cases H : ∃ s : Finset M, Nonempty (Basis s R M)
-  · rcases H with ⟨s, ⟨hs⟩⟩
-    exact Module.Free.of_basis hs
-  · classical simp [LinearMap.coe_det, H] at hf
+@[deprecated (since := "2026-08-29")]
+alias free_of_det_ne_one := Module.Free.of_det_ne_one
 
 /-- If a linear map has determinant different from `1`, then the space is finite-dimensional. -/
 theorem finite_of_det_ne_one {f : M →ₗ[R] M} (hf : f.det ≠ 1) : Module.Finite R M := by
