diff --git a/Mathlib/CategoryTheory/Bicategory/Adjunction/Mate.lean b/Mathlib/CategoryTheory/Bicategory/Adjunction/Mate.lean
index 79257de..32177d8 100644
--- a/Mathlib/CategoryTheory/Bicategory/Adjunction/Mate.lean
+++ b/Mathlib/CategoryTheory/Bicategory/Adjunction/Mate.lean
@@ -508,16 +508,18 @@ lemma conjugateEquiv_whiskerRight
     leftUnitor_inv_whiskerRight, Iso.inv_hom_id, triangle_assoc_comp_right_assoc] at this
   simp [conjugateEquiv_apply, this]
 
-set_option linter.flexible false in -- simp followed by bicategory
 lemma conjugateEquiv_associator_hom
     {a b c d : B} {l₁ : a ⟶ b} {r₁ : b ⟶ a} (adj₁ : l₁ ⊣ r₁)
     {l₂ : b ⟶ c} {r₂ : c ⟶ b} (adj₂ : l₂ ⊣ r₂)
     {l₃ : c ⟶ d} {r₃ : d ⟶ c} (adj₃ : l₃ ⊣ r₃) :
     conjugateEquiv (adj₁.comp (adj₂.comp adj₃))
       ((adj₁.comp adj₂).comp adj₃) (α_ _ _ _).hom = (α_ _ _ _).hom := by
-  simp [← cancel_epi (ρ_ ((r₃ ≫ r₂) ≫ r₁)).hom, ← cancel_mono (λ_ (r₃ ≫ r₂ ≫ r₁)).inv,
-    conjugateEquiv_apply, mateEquiv_eq_iff, Adjunction.homEquiv₁_symm_apply,
-    Adjunction.homEquiv₂_apply]
+  simp only [conjugateEquiv_apply, ← cancel_epi (ρ_ ((r₃ ≫ r₂) ≫ r₁)).hom, Iso.hom_inv_id_assoc,
+    ← cancel_mono (λ_ (r₃ ≫ r₂ ≫ r₁)).inv, Category.assoc, Iso.hom_inv_id, Category.comp_id,
+    mateEquiv_eq_iff, Adjunction.homEquiv₁_symm_apply, unitors_inv_equal, Adjunction.comp_unit,
+    Adjunction.compUnit, whiskerRight_id, whiskerLeft_comp, whiskerLeft_rightUnitor,
+    comp_whiskerLeft, Iso.inv_hom_id_assoc, whiskerLeft_inv_hom_assoc, Adjunction.homEquiv₂_apply,
+    whiskerRight_comp, id_whiskerLeft, unitors_equal, comp_whiskerRight, leftUnitor_whiskerRight]
   bicategory
 
 end conjugateEquiv
