diff --git a/Mathlib/AlgebraicGeometry/AffineScheme.lean b/Mathlib/AlgebraicGeometry/AffineScheme.lean
index dd60bd150..b8147105f 100644
--- a/Mathlib/AlgebraicGeometry/AffineScheme.lean
+++ b/Mathlib/AlgebraicGeometry/AffineScheme.lean
@@ -749,7 +749,7 @@ include hU in
 theorem basicOpen_basicOpen_is_basicOpen (g : Γ(X, X.basicOpen f)) :
     ∃ f' : Γ(X, U), X.basicOpen f' = X.basicOpen g := by
   have := isLocalization_basicOpen hU f
-  obtain ⟨x, ⟨_, n, rfl⟩, rfl⟩ := IsLocalization.surj'' (Submonoid.powers f) g
+  obtain ⟨x, ⟨_, n, rfl⟩, rfl⟩ := IsLocalization.surj' (Submonoid.powers f) g
   use f * x
   rw [Algebra.smul_def, Scheme.basicOpen_mul, Scheme.basicOpen_mul, RingHom.algebraMap_toAlgebra,
     Scheme.basicOpen_res]
diff --git a/Mathlib/Analysis/Complex/Schwarz.lean b/Mathlib/Analysis/Complex/Schwarz.lean
index cef97c3aa..47bc7e636 100644
--- a/Mathlib/Analysis/Complex/Schwarz.lean
+++ b/Mathlib/Analysis/Complex/Schwarz.lean
@@ -327,11 +327,8 @@ if there exists a point `z₀` in the ball such that `‖dslope f c z₀‖ = R
 then the map `f` is affine with the absolute value of the slope equal to `R₂ / R₁`.
 
 This is an existence version of `affine_of_mapsTo_ball_of_norm_dslope_eq_div` above.
-
-TODO: once the deprecated alias `affine_of_mapsTo_ball_of_exists_norm_dslope_eq_div` is gone,
-rename this theorem to `affine_of_mapsTo_ball_of_exists_norm_dslope_eq_div`.
 -/
-theorem affine_of_mapsTo_ball_of_exists_norm_dslope_eq_div'
+theorem affine_of_mapsTo_ball_of_exists_norm_dslope_eq_div
     [StrictConvexSpace ℝ E] (hd : DifferentiableOn ℂ f (ball c R₁))
     (h_maps : Set.MapsTo f (ball c R₁) (closedBall (f c) R₂))
     (h_z₀ : ∃ z₀ ∈ ball c R₁, ‖dslope f c z₀‖ = R₂ / R₁) :
@@ -339,6 +336,10 @@ theorem affine_of_mapsTo_ball_of_exists_norm_dslope_eq_div'
   let ⟨z₀, h_z₀, h_eq⟩ := h_z₀
   ⟨dslope f c z₀, h_eq, affine_of_mapsTo_ball_of_norm_dslope_eq_div hd h_maps h_z₀ h_eq⟩
 
+@[deprecated (since := "2026-08-18")]
+alias affine_of_mapsTo_ball_of_exists_norm_dslope_eq_div' :=
+  affine_of_mapsTo_ball_of_exists_norm_dslope_eq_div
+
 end DimOne
 
 end -- public section
diff --git a/Mathlib/RingTheory/Localization/InvSubmonoid.lean b/Mathlib/RingTheory/Localization/InvSubmonoid.lean
index d6b42c75f..5713f9c33 100644
--- a/Mathlib/RingTheory/Localization/InvSubmonoid.lean
+++ b/Mathlib/RingTheory/Localization/InvSubmonoid.lean
@@ -77,14 +77,14 @@ theorem smul_toInvSubmonoid (m : M) : m • (toInvSubmonoid M S m : S) = 1 := by
 
 variable {S}
 
--- `surj'` was taken, so use `surj''` instead
--- TODO: this can be fixed after the deprecations of 2025-09-04 are removed.
-theorem surj'' (z : S) : ∃ (r : R) (m : M), z = r • (toInvSubmonoid M S m : S) := by
+theorem surj' (z : S) : ∃ (r : R) (m : M), z = r • (toInvSubmonoid M S m : S) := by
   rcases IsLocalization.surj M z with ⟨⟨r, m⟩, e : z * _ = algebraMap R S r⟩
   refine ⟨r, m, ?_⟩
   rw [Algebra.smul_def, ← e, mul_assoc]
   simp
 
+@[deprecated (since := "2026-08-18")] alias surj'' := surj'
+
 theorem toInvSubmonoid_eq_mk' (x : M) : (toInvSubmonoid M S x : S) = mk' S 1 x := by
   rw [← (IsLocalization.map_units S x).mul_left_inj]
   simp
@@ -100,7 +100,7 @@ variable (S)
 theorem span_invSubmonoid : Submodule.span R (invSubmonoid M S : Set S) = ⊤ := by
   rw [eq_top_iff]
   rintro x -
-  rcases IsLocalization.surj'' M x with ⟨r, m, rfl⟩
+  rcases IsLocalization.surj' M x with ⟨r, m, rfl⟩
   exact Submodule.smul_mem _ _ (Submodule.subset_span (toInvSubmonoid M S m).prop)
 
 theorem finiteType_of_monoid_fg [Monoid.FG M] : Algebra.FiniteType R S := by
