diff --git a/Mathlib/Topology/ContinuousOn.lean b/Mathlib/Topology/ContinuousOn.lean
index e46b5f68..873802fa 100644
--- a/Mathlib/Topology/ContinuousOn.lean
+++ b/Mathlib/Topology/ContinuousOn.lean
@@ -891,9 +891,6 @@ theorem continuousOn_union_iff_of_isClosed {f : α → β} (hs : IsClosed s) (ht
   ⟨fun h ↦ ⟨h.mono s.subset_union_left, h.mono s.subset_union_right⟩,
    fun h ↦ h.left.union_of_isClosed h.right hs ht⟩
 
-@[deprecated (since := "2026-02-20")]
-alias continouousOn_union_iff_of_isClosed := continuousOn_union_iff_of_isClosed
-
 /-- If a function is continuous on two open sets, it is also continuous on their union. -/
 theorem ContinuousOn.union_of_isOpen {f : α → β} (hfs : ContinuousOn f s) (hft : ContinuousOn f t)
     (hs : IsOpen s) (ht : IsOpen t) : ContinuousOn f (s ∪ t) :=
@@ -905,9 +902,6 @@ theorem continuousOn_union_iff_of_isOpen {f : α → β} (hs : IsOpen s) (ht : I
   ⟨fun h ↦ ⟨h.mono s.subset_union_left, h.mono s.subset_union_right⟩,
    fun h ↦ h.left.union_of_isOpen h.right hs ht⟩
 
-@[deprecated (since := "2026-02-20")]
-alias continouousOn_union_iff_of_isOpen := continuousOn_union_iff_of_isOpen
-
 /-- If a function is continuous on open sets `s i`, it is continuous on their union -/
 lemma ContinuousOn.iUnion_of_isOpen {ι : Type*} {s : ι → Set α}
     (hf : ∀ i : ι, ContinuousOn f (s i)) (hs : ∀ i, IsOpen (s i)) :
