diff --git a/Mathlib/Analysis/Complex/MeanValue.lean b/Mathlib/Analysis/Complex/MeanValue.lean
index c7a04b9b..2dfe2c02 100644
--- a/Mathlib/Analysis/Complex/MeanValue.lean
+++ b/Mathlib/Analysis/Complex/MeanValue.lean
@@ -77,9 +77,6 @@ theorem DiffContOnCl.circleAverage_smul_div (hf : DiffContOnCl ℂ f (ball c |R|
     rw [sdiff_empty] at hz
     apply (hf.1 z hz).differentiableAt (isOpen_ball.mem_nhds hz)
 
-@[deprecated (since := "2026-02-11")]
-alias circleAverage_sub_sub_inv_smul_of_differentiable_on := DiffContOnCl.circleAverage_smul_div
-
 /-!
 ## Classic Mean Value Properties
 
@@ -116,6 +113,3 @@ theorem DiffContOnCl.circleAverage (hf : DiffContOnCl ℂ f (ball c |R|)) :
     apply circleAverage_congr_sphere fun z hz ↦ ?_
     have : z - c ≠ 0 := by grind [ne_of_mem_sphere]
     simp_all
-
-@[deprecated (since := "2026-02-11")]
-alias circleAverage_of_differentiable_on := DiffContOnCl.circleAverage
