diff --git a/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean b/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean
index 6e3f625..50b66f4 100644
--- a/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean
+++ b/Mathlib/Analysis/SpecialFunctions/Pow/Real.lean
@@ -793,6 +793,10 @@ lemma le_pow_of_log_le (hy : 0 < y) (h : log x ≤ n * log y) : x ≤ y ^ n :=
 lemma le_zpow_of_log_le {n : ℤ} (hy : 0 < y) (h : log x ≤ n * log y) : x ≤ y ^ n :=
   rpow_intCast _ _ ▸ le_rpow_of_log_le hy h
 
+@[deprecated (since := "2026-08-29")] alias rpow_le_of_le_log := le_rpow_of_log_le
+@[deprecated (since := "2026-08-29")] alias pow_le_of_le_log := le_pow_of_log_le
+@[deprecated (since := "2026-08-29")] alias zpow_le_of_le_log := le_zpow_of_log_le
+
 theorem lt_rpow_iff_log_lt (hx : 0 < x) (hy : 0 < y) : x < y ^ z ↔ log x < z * log y := by
   rw [← log_lt_log_iff hx (rpow_pos_of_pos hy z), log_rpow hy]
 
@@ -813,6 +817,10 @@ lemma lt_pow_of_log_lt (hy : 0 < y) (h : log x < n * log y) : x < y ^ n :=
 lemma lt_zpow_of_log_lt {n : ℤ} (hy : 0 < y) (h : log x < n * log y) : x < y ^ n :=
   rpow_intCast _ _ ▸ lt_rpow_of_log_lt hy h
 
+@[deprecated (since := "2026-08-29")] alias rpow_lt_of_lt_log := lt_rpow_of_log_lt
+@[deprecated (since := "2026-08-29")] alias pow_lt_of_lt_log := lt_pow_of_log_lt
+@[deprecated (since := "2026-08-29")] alias zpow_lt_of_lt_log := lt_zpow_of_log_lt
+
 lemma rpow_le_iff_le_log (hx : 0 < x) (hy : 0 < y) : x ^ z ≤ y ↔ z * log x ≤ log y := by
   rw [← log_le_log_iff (rpow_pos_of_pos hx _) hy, log_rpow hx]
 
@@ -831,17 +839,6 @@ lemma le_log_of_pow_le (hx : 0 < x) (h : x ^ n ≤ y) : n * log x ≤ log y :=
 lemma le_log_of_zpow_le {n : ℤ} (hx : 0 < x) (h : x ^ n ≤ y) : n * log x ≤ log y :=
   le_log_of_rpow_le hx (rpow_intCast _ _ ▸ h)
 
-lemma rpow_le_of_le_log (hy : 0 < y) (h : log x ≤ z * log y) : x ≤ y ^ z := by
-  obtain hx | hx := le_or_gt x 0
-  · exact hx.trans (rpow_pos_of_pos hy _).le
-  · exact (le_rpow_iff_log_le hx hy).2 h
-
-lemma pow_le_of_le_log (hy : 0 < y) (h : log x ≤ n * log y) : x ≤ y ^ n :=
-  rpow_natCast _ _ ▸ rpow_le_of_le_log hy h
-
-lemma zpow_le_of_le_log {n : ℤ} (hy : 0 < y) (h : log x ≤ n * log y) : x ≤ y ^ n :=
-  rpow_intCast _ _ ▸ rpow_le_of_le_log hy h
-
 lemma rpow_lt_iff_lt_log (hx : 0 < x) (hy : 0 < y) : x ^ z < y ↔ z * log x < log y := by
   rw [← log_lt_log_iff (rpow_pos_of_pos hx _) hy, log_rpow hx]
 
@@ -860,17 +857,6 @@ lemma lt_log_of_pow_lt (hx : 0 < x) (h : x ^ n < y) : n * log x < log y :=
 lemma lt_log_of_zpow_lt {n : ℤ} (hx : 0 < x) (h : x ^ n < y) : n * log x < log y :=
   lt_log_of_rpow_lt hx (rpow_intCast _ _ ▸ h)
 
-lemma rpow_lt_of_lt_log (hy : 0 < y) (h : log x < z * log y) : x < y ^ z := by
-  obtain hx | hx := le_or_gt x 0
-  · exact hx.trans_lt (rpow_pos_of_pos hy _)
-  · exact (lt_rpow_iff_log_lt hx hy).2 h
-
-lemma pow_lt_of_lt_log (hy : 0 < y) (h : log x < n * log y) : x < y ^ n :=
-  rpow_natCast _ _ ▸ rpow_lt_of_lt_log hy h
-
-lemma zpow_lt_of_lt_log {n : ℤ} (hy : 0 < y) (h : log x < n * log y) : x < y ^ n :=
-  rpow_intCast _ _ ▸ rpow_lt_of_lt_log hy h
-
 theorem rpow_le_one_iff_of_pos (hx : 0 < x) : x ^ y ≤ 1 ↔ 1 ≤ x ∧ y ≤ 0 ∨ x ≤ 1 ∧ 0 ≤ y := by
   rw [rpow_def_of_pos hx, exp_le_one_iff, mul_nonpos_iff, log_nonneg_iff hx, log_nonpos_iff hx.le]
 
