diff --git a/Mathlib/CategoryTheory/Limits/Shapes/Grothendieck.lean b/Mathlib/CategoryTheory/Limits/Shapes/Grothendieck.lean
index 4b389bd..783afe2 100644
--- a/Mathlib/CategoryTheory/Limits/Shapes/Grothendieck.lean
+++ b/Mathlib/CategoryTheory/Limits/Shapes/Grothendieck.lean
@@ -85,8 +85,6 @@ def fiberwiseColimit : C ⥤ H where
 
 set_option backward.defeqAttrib.useBackward true in
 set_option backward.isDefEq.respectTransparency false in
--- TODO: find a good way to fix the linter; simp cannot be combined with the subsequent apply
-set_option linter.flexible false in
 variable (H) (F) in
 /-- Similar to `colimit` and `colim`, taking fiberwise colimits is a functor
 `(Grothendieck F ⥤ H) ⥤ (C ⥤ H)` between functor categories. -/
@@ -96,8 +94,14 @@ def fiberwiseColim [∀ c, HasColimitsOfShape (F.obj c) H] : (Grothendieck F ⥤
   map α :=
     { app := fun c => colim.map (whiskerLeft _ α)
       naturality := fun c₁ c₂ f => by apply colimit.hom_ext; simp }
-  map_id G := by ext; simp; apply Functor.map_id colim
-  map_comp α β := by ext; simp; apply Functor.map_comp colim
+  map_id G := by
+    ext
+    simp only [fiberwiseColimit_obj, whiskerLeft_id', colim_map, NatTrans.id_app]
+    apply Functor.map_id colim
+  map_comp α β := by
+    ext
+    simp only [fiberwiseColimit_obj, whiskerLeft_comp, colim_map, NatTrans.comp_app]
+    apply Functor.map_comp colim
 
 set_option backward.defeqAttrib.useBackward true in
 set_option backward.isDefEq.respectTransparency false in
