diff --git a/Mathlib/Analysis/Polynomial/Basic.lean b/Mathlib/Analysis/Polynomial/Basic.lean
index ca481056..ea47ed65 100644
--- a/Mathlib/Analysis/Polynomial/Basic.lean
+++ b/Mathlib/Analysis/Polynomial/Basic.lean
@@ -38,8 +38,6 @@ variable {𝕜 : Type*} [NormedField 𝕜] [LinearOrder 𝕜] [IsStrictOrderedRi
 theorem eventually_atTop_not_isRoot (hP : P ≠ 0) : ∀ᶠ x in atTop, ¬P.IsRoot x :=
   atTop_le_cofinite <| (finite_setOfPred_isRoot hP).compl_mem_cofinite
 
-@[deprecated (since := "2026-02-05")] alias eventually_no_roots := eventually_atTop_not_isRoot
-
 theorem eventually_atBot_not_isRoot (hP : P ≠ 0) : ∀ᶠ x in atBot, ¬P.IsRoot x :=
   atBot_le_cofinite <| (finite_setOfPred_isRoot hP).compl_mem_cofinite
 
@@ -98,8 +96,6 @@ theorem isBoundedUnder_abs_atTop_iff :
   contrapose! h
   exact not_isBoundedUnder_of_tendsto_atTop (abs_tendsto_atTop P h)
 
-@[deprecated (since := "2026-02-05")] alias abs_isBoundedUnder_iff := isBoundedUnder_abs_atTop_iff
-
 theorem abs_tendsto_atTop_iff : Tendsto (fun x => abs <| eval x P) atTop atTop ↔ 0 < P.degree :=
   ⟨fun h ↦ not_le.mp (mt (isBoundedUnder_abs_atTop_iff P).mpr
     (not_isBoundedUnder_of_tendsto_atTop h)), abs_tendsto_atTop P⟩
@@ -170,9 +166,6 @@ theorem div_tendsto_atTop_zero_of_degree_lt (hdeg : P.degree < Q.degree) :
   refine (tendsto_zpow_atTop_zero ?_).const_mul _
   lia
 
-@[deprecated (since := "2026-02-05")]
-alias div_tendsto_zero_of_degree_lt := div_tendsto_atTop_zero_of_degree_lt
-
 theorem div_tendsto_atTop_zero_iff_degree_lt (hQ : Q ≠ 0) :
     Tendsto (fun x => eval x P / eval x Q) atTop (𝓝 0) ↔ P.degree < Q.degree := by
   refine ⟨fun h => ?_, div_tendsto_atTop_zero_of_degree_lt P Q⟩
@@ -189,18 +182,12 @@ theorem div_tendsto_atTop_zero_iff_degree_lt (hQ : Q ≠ 0) :
     · rw [sub_lt_iff_lt_add, zero_add, Int.ofNat_lt] at h
       exact degree_lt_degree h.1
 
-@[deprecated (since := "2026-02-05")]
-alias div_tendsto_zero_iff_degree_lt := div_tendsto_atTop_zero_iff_degree_lt
-
 theorem div_tendsto_atTop_leadingCoeff_div_of_degree_eq (hdeg : P.degree = Q.degree) :
     Tendsto (fun x => eval x P / eval x Q) atTop (𝓝 <| P.leadingCoeff / Q.leadingCoeff) := by
   refine (isEquivalent_atTop_div P Q).symm.tendsto_nhds ?_
   rw [show (P.natDegree : ℤ) = Q.natDegree by simp [hdeg, natDegree]]
   simp
 
-@[deprecated (since := "2026-02-05")]
-alias div_tendsto_leadingCoeff_div_of_degree_eq := div_tendsto_atTop_leadingCoeff_div_of_degree_eq
-
 theorem div_tendsto_atTop_of_degree_gt' (hdeg : Q.degree < P.degree)
     (hpos : 0 < P.leadingCoeff / Q.leadingCoeff) :
     Tendsto (fun x => eval x P / eval x Q) atTop atTop := by
@@ -249,9 +236,6 @@ theorem abs_div_tendsto_atTop_atTop_of_degree_gt (hdeg : Q.degree < P.degree) (h
   · exact tendsto_abs_atTop_atTop.comp (P.div_tendsto_atTop_of_degree_gt Q hdeg hQ h)
   · exact tendsto_abs_atBot_atTop.comp (P.div_tendsto_atBot_of_degree_gt Q hdeg hQ h.le)
 
-@[deprecated (since := "2026-02-05")]
-alias abs_div_tendsto_atTop_of_degree_gt := abs_div_tendsto_atTop_atTop_of_degree_gt
-
 end PolynomialDivAtTop
 
 section PolynomialDivAtBot
@@ -330,8 +314,6 @@ theorem isBigO_atBot_of_degree_le (h : P.degree ≤ Q.degree) : P.eval =O[atBot]
   convert! (isBigO_atTop_of_degree_le _ _ h).comp_tendsto tendsto_neg_atBot_atTop using 2
   all_goals simp
 
-@[deprecated (since := "2026-02-05")] alias isBigO_of_degree_le := isBigO_atTop_of_degree_le
-
 section Cobounded
 
 open Bornology
